apps/frontend/src/app/3_organisms/Footer/Footer.tsx

```diff
@@ -1,43 +1,60 @@
 import React, { FC } from 'react';
 
+import { useTranslation } from 'react-i18next';
+
 import { Link, Footer as UIFooter } from '@sovryn/ui';
 
 import { SocialLinks } from '../../2_molecules';
 import { SovrynLogo } from '../../2_molecules/SovrynLogo/SovrynLogo';
 import Logo from '../../../assets/images/sovryn-small-logo.svg';
-import { sovrynLinks } from '../../../utils/constants';
+import { translations } from '../../../locales/i18n';
+import { Environments } from '../../../types/global';
+import {
+  sovrynLinks,
+  sovrynAlphaLinks,
+  sovrynWikiLinks,
+} from '../../../utils/constants';
+import { isMainnet } from '../../../utils/helpers';
 
 export const Footer: FC = () => {
+  const { t } = useTranslation();
   const footerLinks = [
```

Comment:
Move `footerLinks` above the component (you don't need to use `useTranslation` hook, just import `{t}` from `i18next` ).

Reply:
awesome, didn't know we could do this :)